rcl/include/rcl/subscription.h

```diff
@@ -307,6 +307,10 @@ RCL_PUBLIC
 RCL_WARN_UNUSED
 const rcl_subscription_options_t *
 rcl_subscription_get_options(const rcl_subscription_t * subscription);
+/// unsafe _subscription_get_options
```

Comment:
Again, here but not every where. Should be removed here or added everywhere else.